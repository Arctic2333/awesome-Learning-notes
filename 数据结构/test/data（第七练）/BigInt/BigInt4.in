10000000000000000
35123312111561175