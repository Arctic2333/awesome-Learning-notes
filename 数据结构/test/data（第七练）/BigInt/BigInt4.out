45123312111561175
